### DISCUSSION TITLE
meeting #1

### DISCUSSION THREAD

**Discussion by shenyih0ng** [usdevs/laundro#2] [Meeting Notes]
# Meeting 1

- [ ] would be good if we can get a recording of how the interface works during a full wash/dry cycle

**Comment by tanhl2000** [comment]
## Power Sockets

### Washer:
- One RPI to one washing machine powered by each socket 
### Dryer:
-

![power-sockets](https://user-images.githubusercontent.com/24812973/144816873-a0127e9e-3471-4c3b-a6df-5f05286c9809.jpg)
![double-socket](https://user-images.githubusercontent.com/24812973/144816896-00ee43d9-ea8c-4673-90fb-7d11485ea21c.jpg)

**Comment by shenyih0ng** [comment]
## Washer and Dryer Interfaces

![washer-interface](https://user-images.githubusercontent.com/24812973/144816886-91128441-5c61-4acb-9e64-742a4059e9cc.jpg)
![dry-interface](https://user-images.githubusercontent.com/24812973/144816892-1763bef7-ebe7-4d38-a164-d497bf73be0a.jpg)

### Door lock light:

Detecting the state of the washing machines through sensing the door lock light. Don't need to account for the blinking of the process lights. Need more observation of the behavior of the door lock light during various stages of the washing process to assess viability.

### Start light:

Using the light from the start button to detect the washing machine state is viable as it will only be on while it is washing (need check if will reset after it is done washing).

**Reply by SeekSaveServe** [reply]
More on the Door Lock Light:

- Door lock light is disarmed after the washing is paused mid-washing and is armed again after the washing is resumed 
- Suggestion is to only consider washing as done if the door lock light has been switched off for more than 1-2 mins

**Comment by shenyih0ng** [comment]
## Circuit Layouts

![circuit-layout1](https://user-images.githubusercontent.com/24812973/144816879-838c35a4-750a-49a7-ab50-30d1ab162b46.jpg)
![circuit-layout2](https://user-images.githubusercontent.com/24812973/144816882-c9d47547-feca-4515-b96f-9ef79ede9a25.jpg)
